admin-dev/themes/new-theme/public/catalog_product.bundle.js

```diff
@@ -1,3 +1,3 @@
e(translate_javascripts["This will delete the specific price. Do you wish to proceed?"],null,{onContinue:function(){var t=o(e).attr("href");o(e).attr("disabled","disabled"),o.ajax({type:"GET",url:t}).done(function(t){i.loadAndDisplayExistingSpecificPricesList(),showSuccessMessage(t),o(e).removeAttr("disabled")}).fail(function(i){showErrorMessage(i.responseJSON),o(e).removeAttr("disabled")})}}).show()}},{key:"storePriceFormDefaultValues",value:function(){var e=this.$createPriceFormDefaultValues;o("#specific_price_form").find("select,input").each(function(i,t){e[o(t).attr("id")]=o(t).val()}),o("#specific_price_form").find("input:checkbox").each(function(i,t){e[o(t).attr("id")]=o(t).prop("checked")}),this.$createPriceFormDefaultValues=e}},{key:"loadAndFillOptionsForSelectCombinationInput",value:function(e){var i=this.getPrefixSelector(e),t=o(i+"sp_id_product_attribute"),r=t.attr("data-action").replace(/product-combinations\/\d+/,"product-combinations/"+this.getProductId());o.ajax({type:"GET",url:r}).done(function(e){t.find("option:gt(0)").remove(),o.each(e,function(e,i){t.append('<option value="'+i.id+'">'+i.name+"</option>")}),"0"!=t.data("selectedAttribute")&&t.val(t.data("selectedAttribute")).trigger("change")})}},{key:"enableSpecificPriceTaxFieldIfEligible",value:function(e){var i=this.getPrefixSelector(e);"percentage"===o(i+"sp_reduction_type").val()?o(i+"sp_reduction_tax").hide():o(i+"sp_reduction_tax").show()}},{key:"resetCreatePriceFormDefaultValues",value:function(){var e=this.$createPriceFormDefaultValues;o("#specific_price_form").find("input").each(function(i,t){o(t).val(e[o(t).attr("id")])}),o("#specific_price_form").find("select").each(function(i,t){o(t).val(e[o(t).attr("id")]).change()}),o("#specific_price_form").find("input:checkbox").each(function(e,i){o(i).prop("checked",!0)})}},{key:"enableSpecificPriceFieldIfEligible",value:function(e){var i=this.getPrefixSelector(e);o(i+"sp_price").prop("disabled",o(i+"leave_bprice").is(":checked")).val("")}},{key:"openEditPriceModalAndLoadForm",value:function(e){var i=this,t=o("#js-specific-price-list").data("actionEdit").replace(/form\/\d+/,"form/"+e);o("#edit-specific-price-modal").modal("show"),this.editModalIsOpen=!0,o.ajax({type:"GET",url:t}).done(function(t){i.insertEditSpecificPriceFormIntoModal(t),o("#edit-specific-price-modal-form").data("specificPriceId",e),i.configureEditPriceFormInsideModalBehavior()}).fail(function(e){showErrorMessage(e.responseJSON)})}},{key:"closeEditPriceModalAndRemoveForm",value:function(){o("#edit-specific-price-modal").modal("hide"),this.editModalIsOpen=!1,o("#edit-specific-price-modal-form").empty()}},{key:"insertEditSpecificPriceFormIntoModal",value:function(e){var i=o("#edit-specific-price-modal-form");i.empty(),i.append(e)}},{key:"getProductId",value:function(){return o("#form_id_product").val()}},{key:"getPrefixSelector",value:function(e){return 1==e?"#"+this.prefixCreateForm:"#"+this.prefixEditForm}}]),e}();i.a=a},409:function(e,i,t){e.exports=t(197)}});
```

Comment:
It seems that there are a few wordings here... should I check them and indicate domains?

Reply:
nope, there are not wording in this PR.

Reply:
Nice, thanks for the answer!